client/agent-opentracing/src/main/resources/apmrules/org/jboss/apm-jboss-ee.btm

```diff
@@ -0,0 +1,23 @@
+# Exclude 'onMessage' method as will conflict with instrumentation of JMS/MDB
+
+RULE org.jboss.ee(1) EE Component Start
+CLASS org.jboss.as.ee.component.ProxyInvocationHandler
+METHOD invoke(Object,Method,Object[])
+HELPER org.hawkular.apm.agent.opentracing.OpenTracingManager
+AT ENTRY
+IF !$2.getName().equals("onMessage")
+DO
+  startSpanWithParent(getTracer().buildSpan($2.getName())
+  		.withTag("EJB.uri", truncateAtDelimiter(simpleClassName($1),"$$$")),
```

Comment:
Its the implementing class name. I'll investigate, I originally copied some byteman rules someone else had created for tracking EJB usage.